cirq/ops/sycamore_gate.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An instance of FSimGate that works naturally on Google's Sycamore chip"""
+
+from typing import TYPE_CHECKING
+
+import numpy as np
+
+from cirq import ops, protocols
+
+if TYPE_CHECKING:
+    import cirq
+
+
+class SycamoreGate(ops.FSimGate):
+    """The Sycamore gate is a two-qubit operation equivalent to
+    FSimGate(π/2, π/6).
+
+    The unitary of this gate is
+
+        [[1, 0, 0, 0],
+         [0, 0, -1j, 0],
+         [0, -1j, 0, 0],
+         [0, 0, 0, exp(- 1j * π/6)]]
+
+    This gate can be performed on the Sycamore chip by Google and
+    was used to demonstrate quantum supremacy as can be seen in
```

Comment:
This should probably not say that this gate was used, since we used variations of this gate (differ across the device).  So possibly could be

"This gate can be performed on the Google's Sycamore chip and
is close to the gates that were used to demonstrate quantum 
supremacy used in this paper: 
https://www.nature.com/articles/s41586-019-1666-5."